widget/src/components/messages/FileMessage.tsx

```diff
@@ -23,19 +22,36 @@ const FileMessage: React.FC<FileMessageProps> = ({ message }) => {
   const { t } = useTranslation();
   const { colors: allColors } = useColors();
   const colors = allColors[message.direction || Direction.received];
+  const [videoErrored, setVideoErrored] = useState(false);
+  const [audioErrored, setAudioErrored] = useState(false);
+  const [imageErrored, setImageErrored] = useState(false);
 
   if (!("type" in message.data)) {
     throw new Error("Unable to detect type for file message");
   }
-
+  if (message.data.type === "unknown") {
+    return (
+      <div
+        className="sc-message--file"
+        style={{
+          color: colors.text,
+          backgroundColor: colors.bg,
+        }}
+      >
+        <p className="error-message">
+          {t("messages.file_message.unsupported_file_type")}
+        </p>
+      </div>
+    );
+  }
```

Comment:
Hi @abdou6666, I suggest to create a function to handle the message.data.type values.
The intent is to centralize the check in one place.
Please, don't hesitate to ask if you have any questions.